Formula/axel.rb

```diff
@@ -12,13 +12,15 @@ class Axel < Formula
     sha256 "b0a7bcbab57df1b9877ac3a6a8343a3a5b4256b4cfa5a6bfe777b9ba92930d31" => :mavericks
   end
 
-  def install
-    system "./configure", "--prefix=#{prefix}",
-                          "--etcdir=#{etc}",
-                          "--debug=0",
-                          "--i18n=0"
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gettext"
+  depends_on "openssl" => :recommended
+  depends_on "libressl" => :optional
 
-    system "make"
+  def install
+    system "./autogen.sh"
+    system "./configure", "--prefix=#{prefix}", "--mandir=#{man}"
```

Comment:
That smells like we might still need `sysconfidir`, since we don't want configuration files written inside the `prefix`?